src/exercise/client/src/App.js

```diff
@@ -2,59 +2,32 @@ import "./App.css";
 import { DialogContentContainer } from "monday-ui-react-core";
 import "monday-ui-react-core/dist/main.css";
 import Header from "./Components/Header/Header";
-import InputItem from "./Components/InputItem/InputItem";
-import BottomButtons from "./Components/BottomButtons/BottomButtons";
-import { useCallback, useEffect, useState, useRef } from "react";
-import TaskList from "./Components/TaskList/TaskList";
-import {
-  getTasks,
-  deleteTask,
-  changeTaskStatus,
-  addTask,
-} from "./service/item_client";
+import InputItem from "./Components/InputItem/InputItem-connector";
+import BottomButtons from "./Components/BottomButtons/BottomButtons-connector";
+import TaskList from "./Components/TaskList/TaskList-connector";
+import { useState, useRef } from "react";
```

Comment:
styling - it's better to perform imports in the following order
react & core libraries
other npm packages
files from relative paths